task_11/app.js

```diff
@@ -0,0 +1,3 @@
+const car = 'Porsche Taycan'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887247)
```suggestion
const car = "Porsche Taycan"
```